app/service.py

```diff
@@ -215,31 +215,31 @@ def verify_answer(answerID, user_data):
     })
 
 def _ensure_user(user_data):
-    user = User.query.filter_by(messenger_id=user_data.messenger_user_id).first()
+    user = User.query.filter_by(messenger_id=user_data['messenger user id']).first()
     if not user:
-        user = User(user_data.messenger_user_id)
+        user = User(user_data['messenger user id'])
```

Comment:
Mozna jenom neznam dostatecne tuhle domenu, ale jestlize usera nemame v databazi, jak muze v requestu znat svoje id?

Reply:
To messenger user id je něco, co Chatfuel chatbot umí posílat, když se to tak v daném bloku nastaví. A je to uživatelovo Messenger ID, ne naše DB ID